src/platform/site-wide/sass/overrides/uswds/focus.scss

```diff
@@ -0,0 +1,138 @@
+@mixin focus-gold-light-outline {
+  outline: 2px solid $color-gold-light;
+  outline-offset: 2px;
+}
+
+@mixin focus-gold-lighter-highlight {
+  background: $color-gold-lighter;
+  outline: 2px solid $color-gold-lighter;
+  outline-offset: 0;
+}
+
+a:focus {
+  @include focus-gold-light-outline;
+  outline-offset: 0;
+}
+
+input:focus {
+  @include focus-gold-light-outline;
+}
+
+select:focus {
+  @include focus-gold-light-outline;
+}
+
+.progress-bar-segmented:focus {
```

Comment:
It seems like most of the rules in here should live in (or near) the files where the other style rules for these classes are.